README.md

```diff
@@ -1,50 +1,50 @@
+The autocomplete uses data from the UK government's country and territory registers. To configure the data used in the autocomplete to suit the specific needs of your product or service, please [contact the registers team](https://registers.cloudapps.digital/support).
 
-The picker itself follows the common look and feel of GOV.UK in line with the [design principles](https://www.gov.uk/design-principles).
+The autocomplete itself follows the common look and feel of GOV.UK in line with the [design principles](https://www.gov.uk/design-principles).
 
-![A screenshot of the new country and territory picker](docs/location-picker-general.gif)
+![A screenshot of the new country and territory autocomplete](docs/location-picker-general.gif)
 
 This guide will show you how to:
-* populate the picker's autocomplete field
-* use the picker's data file
+* populate the autocomplete field
+* use the autocomplete's data file
 
 [Contact the registers team](https://registers.cloudapps.digital/support) if you have any problems or questions that are not covered in this guide.
 
-[Try out the example.](https://alphagov.github.io/openregister-location-picker/examples/)
+[Try out the example.](https://alphagov.github.io/govuk-country-territory-autocomplete/examples/)
 
 ## Before you start
 
 This guide assumes you have a service that outputs HTML. It makes some choices for you with respect to which data is passed back and persisted to your service. Please [contact the registers team](https://registers.cloudapps.digital/support) to discuss your requirements.
 
 ## Integration process
 
-To integrate an application with the picker, you'll need to:
+To integrate an application with the autocomplete, you'll need to:
 * use the data from the country and territory registers
 * create an accessible autocomplete widget
 * keep the data up to date
 
 ### Add location data from country and territory registers
 
-To use register data in the picker, you will need two files:
+To use register data in the autocomplete, you will need two files:
 * [location-picker-canonical-list.json](dist/location-picker-canonical-list.json) - a list of every canonical country and territory
 * [location-picker-graph.json](dist/location-picker-graph.json) - a directed acyclic graph mapping canonical locations to abbreviations, synonyms, endonyms, and typos
 
 The `location-picker-graph.json` file only contains examples of synonyms, abbreviations, endonyms and typos you might want to consider. It is not a comprehensive list. You may wish to add or remove items based on your own user research.
 
 Copy both files to your application. The `location-picker-graph.json` file must be exposed as a public asset.
 
-You can also install the location picker using `npm`:
+You can also install the location autocomplete using `npm`:
 
 ```bash
-$ npm install openregister-location-picker
```

Comment:
Do we need to republish this to npm? Is there a concept of a redirect in npm?

Reply:
I think so @nickcolley, yes. Can you help? 

https://www.npmjs.com/package/openregister-location-picker

Reply:
The patterns and tools team are publishing GOV.UK stuff to npm, be good to check in with them 👍

Reply:
Think we'll need to deprecate the current one and then publish the other as per pattern and tools advice.

https://docs.npmjs.com/cli/deprecate

Reply:
Not quiet as elegant as Github but yes, I agree

Reply:
Going to revert this change until we publish a new package

Reply:
Actually we are going to publish a new package to npm now 👍

Reply:
Sorted @nickcolley 

```
$ npm install openregister-location-picker
npm WARN deprecated openregister-location-picker@0.3.2: Move to govuk-country-and-territory-autocomplete
```

Reply:
Nice work @danmitchell- 🙌